lib/rspec-steps/hook.rb

```diff
@@ -0,0 +1,34 @@
+require 'rspec-steps/dsl'
+
+module RSpec::Steps
+  class Hook < Struct.new(:type, :kind, :action)
+    def rspec_kind
+      case kind
+      when :each
+        warn_about_promotion(type)
+        :all
+      when :step
+        :each
+      else
+        kind
+      end
+    end
+
+    def warn_about_promotion(scope_name)
+      RSpec::Steps.warnings[
+        "#{scope_name} :each blocks declared for steps are always treated as " +
```

Comment:
Readme seems to suggest before :step has gone away, but this message suggests otherwise. Which is correct?